We've seen some tests take more than 20s to complete on a busy armhf
machine. Even a busy x86_64 machine can use more than 5s on some tests.

Increase timeouts to increase chances of a successful build.

--- a/nss/gtests/ssl_gtest/tls_connect.cc	2017-03-14 22:47:30.855813629 +0100
+++ b/nss/gtests/ssl_gtest/tls_connect.cc	2017-03-14 22:48:49.042335273 +0100
@@ -245,7 +245,7 @@
 
   ASSERT_TRUE_WAIT((client_->state() != TlsAgent::STATE_CONNECTING) &&
                        (server_->state() != TlsAgent::STATE_CONNECTING),
-                   5000);
+                   25000);
 }
 
 void TlsConnectTestBase::EnableExtendedMasterSecret() {
@@ -385,7 +385,7 @@
   if (failing_side == TlsAgent::CLIENT) {
     failing_agent = client_;
   }
-  ASSERT_TRUE_WAIT(failing_agent->state() == TlsAgent::STATE_ERROR, 5000);
+  ASSERT_TRUE_WAIT(failing_agent->state() == TlsAgent::STATE_ERROR, 25000);
 }
 
 void TlsConnectTestBase::ConfigureVersion(uint16_t version) {
